src/styles.ts

```diff
@@ -35,16 +35,17 @@ export const styles = css`
 			--cosmoz-input-focused-color,
 			var(--primary-color, #3f51b5)
 		);
+		--cosmoz-input-border-color: var(--line-color);
```

Comment:
If you set `--cosmoz-input-border-color` here, you are making it impossible to change the value via the CSS cascade, forcing authors to have to set the `--cosmoz-input-border-color` directly on the `<cosmoz-input>` element to override the value. If it's not meant to be a public variable, it is fine, but you should rename is as `--border-color`, the `--cosmoz-input-whatever` is meant for public variables.

Reply:
Understood. Also, I noticed that there are two definition of that var - at line 38 and 41, one linked to the `--line-color` and one to the `--contour-color`, which one should I keep?

Reply:
Is the var used anywhere? I can't find it's usage. If it's not used, drop it.

Reply:
On second thought @cristinecula I don't think it plays a huge role which one I delete, since on the line above, the value of `--contour-color` is set equal to `--line-color` ...

I deleted the doubling, I'm pushing the latest changes.'